src/vtt-segment-loader.js

```diff
@@ -334,11 +334,13 @@ export default class VTTSegmentLoader extends SegmentLoader {
       return;
     }
 
-    this.updateTimeMapping_(
-      segmentInfo,
-      this.syncController_.timelines[segmentInfo.timeline],
-      this.playlist_
-    );
+    if (!isMp4WebVttSegment) {
```

Comment:
I'm sure there is a good reason, but why do we not want to update the time mapping in this case?

Reply:
The `updateTimeMapping_` function being called here is specific to MPEGTS/vtt time mapping. Since these are mp4 segments we use the parent segment loader timing.